identity/src/commonMain/kotlin/com/android/identity/nfc/NfcTagLostException.kt

```diff
@@ -0,0 +1,35 @@
+package com.android.identity.nfc
+
+/**
+ * Exception thrown if a [NfcIsoTag] is removed while trying to communicate with it.
+ */
+class NfcTagLostException: Exception {
+    /**
+     * Construct a new exception.
+     */
+    constructor()
```

Comment:
Remove the parameterless constructor? Seems like we always want at least some sort of error message.

Reply:
That's fair. Done.